deploy/operator/base/rbac.yaml

```diff
@@ -97,6 +97,18 @@ rules:
       - update
       - list
       - watch
+  - apiGroups:
+    - ""
+    resources:
+      - configmaps
+    verbs:
+      - get
+      - create
+      - patch
+      - list
+      - watch
```

Comment:
why does the controller need watch here since it is just calling get, especially if we add configmap to the list of types we disable for the controller runtime cache like https://github.com/redhat-appstudio/jvm-build-service/blob/c4ae49bc3f44458a8b240251c08064139557cfd9/pkg/controller/controller.go#L79-L85

so long term if we will use the pipeline SA for the cache, since RebuiltArtifacts.java does the watch, we just need to make sure it can do the CM watch. which it might have already.  And if it doesn't, lets create a new role / rolebinding that we aggregate to edit, to the pipeline SA gets it that way